### PR TITLE
[RFC][dkr] clean up image names

### PR DESCRIPTION
**Pull request by sausagee**
## Motivation
Conform and clean up image names. Ex libra_validator, libra_e2e, validator --> libra/validator

## Test Plan
To be canaried e2e.

### CODE REVIEW

File: docker/mint/build.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@ set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
-$DIR/../libra-build.sh $DIR/Dockerfile libra_mint "$@"
+$DIR/../libra-build.sh $DIR/Dockerfile libra/mint "$@"
```
Comment (rexhoffman) [inline docker/mint/build.sh:8]: We can rename this to libra/faucet

File: docker/safety-rules/build.sh [hunk 2]
```diff
@@ -5,4 +5,4 @@ set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
-$DIR/../libra-build.sh $DIR/Dockerfile libra_safety_rules "$@"
+$DIR/../libra-build.sh $DIR/Dockerfile libra/safety_rules "$@"
```
Comment (rexhoffman) [inline docker/safety-rules/build.sh:8]: now this should be libra/validator_tcb

File: docker/validator-dynamic/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ### Production Image ###
-FROM libra_e2e:latest as validator_with_config
+FROM libra/validator:latest as validator_with_config
```
Comment (rexhoffman) [inline docker/validator-dynamic/Dockerfile:2]: @davidiw is deleting this.

File: docker/validator-dynamic/build.sh [hunk 4]
```diff
@@ -5,5 +5,5 @@ set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
-$DIR/../libra-build.sh $DIR/../validator/Dockerfile libra_e2e "$@"
-$DIR/../libra-build.sh $DIR/Dockerfile libra_validator_dynamic
+$DIR/../libra-build.sh $DIR/../validator/Dockerfile libra/validator "$@"
+$DIR/../libra-build.sh $DIR/Dockerfile libra/validator_dynamic
```
Comment (rexhoffman) [inline docker/validator-dynamic/build.sh:9]: and deleting this.

File: docker/safety-rules/build.sh (hunk 2 above)
Comment (davidiw) [reply docker/safety-rules/build.sh:8]: Is the intent that we'll have a single docker container containing all the varieties of tcb? (key-manager, safety-rules, execution -- eventually)

File: docker/validator-dynamic/build.sh (hunk 4 above)
Comment (davidiw) [reply docker/validator-dynamic/build.sh:9]: this one might be tricky without handle a PR for the operational component that is out of scope of this repo :/

**Comment by davidiw** [comment]
feel free to continue on with this, config-builder has been removed!

**Comment by sausagee** [comment]
superseded by #6132

**Pull request closed without merging**